features/example/example.md

```diff
@@ -0,0 +1,25 @@
+# Example feature spec
+This contains an example of a new feature to be added.
```

Comment:
Related to earlier comment... but this line here could be: <A short description of the feature>. And that is what would be in the template. But the example here is what you have now. Same for the sections below. Or it could be merged into this example.